File: 006/3.js

// 3)	Faça um programa que leia números inteiros maiores que zero. Quando for entrado o número zero, o programa deverá apresentar quantos números foram entrados e a média destes.


let number      = 0;
let sumNumbers  = 0;
let cont        = 0;

let allNumbers  = [];

while (true) {
    
    number = parseInt(prompt("Informe um número"));

    if (number > 0) {
        sumNumbers += number;
        allNumbers[cont] = number;
        cont++;
    }
    else {
        if (number == 0) { break; }
    }
}

console.log("Média: "            + (sumNumbers / cont));
console.log("Todos os números: " + allNumbers);

File: ExerciciosOpcionais/01/3.py

# 3)	[ENQUANTO] Crie um programa que leia 6 números inteiros e no final mostre quantos deles são pares e quantos são ímpares.

par     = 0
impar   = 0
rotacao = 0
numero  = 0

while rotacao < 6:

    numero = int(input("Informe o " + str(rotacao+1) + "º número: "))

    if (numero % 2) == 0:
        par += 1
    else:
        impar += 1

    rotacao += 1

print("Pares: "  + str(par))
print("Ímpares:" + str(impar))
